transform/nijz_daily.py

```diff
@@ -177,7 +177,7 @@ def get_region_header(region: str):
         'SKUPAJ': 'cases.confirmed'
     }, axis='columns').set_index('date')
 df_1 = df_1[df_1.index.notnull()]  # drop non-indexed rows (where NaN is a part of the index)
-df_1.drop('SKUPAJ', axis='rows', inplace=True)
+df_1.drop('SKUPAJ', axis='rows', errors='ignore', inplace=True) # in newer panda versions this gets removed by skipfooter=1
```

Comment:
Sicer bi se mi skoraj bolj prav zdelo izbrisati `skipfooter=1` nekaj vrstic višje.

Reply:
ja, možno da pri prejšnih verzijah skipfooter ni delal prav, zdaj so pa to popravili. Se strinjam da je bolje da na roke dropnemo in umaknemo skipfooter.

Reply:
Ne, `skipfooter` dela prav. Samo panda "iz lufta" potegne eno prazno vrstico na koncu datoteke, in tisti, ki je pisal kodo, je dodal dva workarounda (ki naredita isto) v izogib (po mojem mnenju) buga v pandi. En workaround bo dovolj, bom spremenila.

Reply:
Tole je zdaj OK